src/Options.php

```diff
@@ -619,6 +619,28 @@ public function setMaxValueLength(int $maxValueLength): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * Gets the http proxy setting.
+     *
+     * @return string
```

Comment:
I don't know why PHPStan doesn't report this, but the docblock is incorrect since the return value can be `null`